src/components/number-picker/number-picker.component.ts

```diff
@@ -107,6 +108,12 @@ export class NumberPickerComponent implements ControlValueAccessor {
         @Optional() public _formGroup: FormGroupDirective
     ) { }
 
+    ngOnInit(): void {
+        if (this.valid) {
```

Comment:
Since `valid` is true by default, this would display for anyone correctly not using the input, which is not desirable.

Perhaps we could use `ngOnChanges` instead, and display the warning on the condition `changes.valid && changes.valid.isFirstChange()`?

https://angular.io/api/core/OnChanges#ngOnChanges